src/Curation/Curation.router.spec.ts

```diff
@@ -229,6 +230,65 @@ describe('when handling a request', () => {
     })
   })
 
+  describe('when trying to obtain a list of item curations', () => {
+    let service: CurationService<any>
+    beforeEach(() => {
+      mockServiceWithAccess(ItemCuration, true)
+      service = mockService(ItemCuration)
+    })
+
+    describe('when itemIds param is not provided', () => {
+      beforeEach(() => {
+        ;(ItemCuration.findByCollectionId as jest.Mock).mockResolvedValueOnce(
+          []
+        )
+      })
+
+      it('should resolve with the collections provided by ItemCuration.findByCollectionId', async () => {
+        const findByCollectionIdSpy = jest
+          .spyOn(ItemCuration, 'findByCollectionId')
+          .mockResolvedValueOnce([])
+
+        const req = {
+          params: { id: 'collectionId' },
+          auth: { ethAddress: 'ethAddress' },
+        } as any
+
+        await router.getCollectionItemCurations(req)
+
+        expect(findByCollectionIdSpy).toHaveBeenCalled()
+      })
+    })
+
+    describe('when itemIds param is provided', () => {
+      let itemIds: string[]
+      beforeEach(() => {
+        itemIds = ['1', '2', '3']
+        ;(ItemCuration.findByCollectionId as jest.Mock).mockResolvedValueOnce(
+          []
+        )
+      })
+
+      it('should resolve with the collections provided by ItemCuration.findByCollectionAndItemIds', async () => {
+        const findByCollectionAndItemIdsSpy = jest
+          .spyOn(ItemCuration, 'findByCollectionAndItemIds')
+          .mockResolvedValueOnce([])
+
+        const req = {
+          params: { id: 'collectionId', itemIds },
+          auth: { ethAddress: 'ethAddress' },
+        } as any
+
+        await router.getCollectionItemCurations(req)
+
+        expect(findByCollectionAndItemIdsSpy).toHaveBeenCalledWith(
+          'collectionId',
+          itemIds
+        )
```

Comment:
I would prefer if instead of checking that the `findByCollectionAndItemIds` method is called, we could check the result of `router.getCollectionItemCurations` was built by mocking `findByCollectionAndItemIds`. If `findByCollectionId` were to be called, as it's not mocked, it won't return what we're expecting.